app/views/home/dashboard.scala.html

```diff
@@ -149,91 +102,88 @@ <h2 class="heading-medium">2. Take the online tests</h2>
                 </div>
             </li>
 
-            <li class="third-step @setVisibility(dashboardPage.thirdStepVisibility)">
+            <li class="third-step @dashboardPage.isThirdStepVisible">
```

Comment:
Not In Jira but we have a confluence page where we document technical debt

https://confluence.tools.tax.service.gov.uk/login.action?os_destination=%2Fpages%2Fviewpage.action%3FspaceKey%3DFSET%26title%3DTechnical%2BDebt&permissionViolation=true